### ISSUE TITLE
XAML designer crashes on MaterialDesignTheme.Defaults.Xaml

### ISSUE THREAD

**Issue by DominikPalo** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#401]
Hello,

I'm using the toolkit for some windows in my WPF app (not for the entire app), but I can't use XAML designer - after adding Material Design resource dictionaries to some window, designer crashes on this window (file `MaterialDesignTheme.Defaults.Xaml`). See screenshot:

![image](https://cloud.githubusercontent.com/assets/1623257/15817527/b4ad7b2c-2bd8-11e6-9893-ba9b57993b88.png)

I'm using current version of toolkit - cloned from the repository and added projects `MaterialDesignThemes.Wpf` and `MaterialDesignColors.Wpf` to my solution (bacause latest published NuGet package missing some new features)

**Comment by ButchersBoy** [comment]
(note if you want leatest features install -pre version from nuget to get 2.0)

**Comment by DominikPalo** [comment]
Thanks for note, I've used wrong package (https://www.nuget.org/packages/MaterialDesign.dll/) without -pre version, now I find and switched to the correct/maintained packed (https://www.nuget.org/packages/MaterialDesignThemes/2.0.0-ci609) 👍  So I finally don't need to include projects to my solutions... but problem with the designer remains

**Comment by ButchersBoy** [comment]
Just started a brand new app on 2.0, and designer is OK if I use resources in the Window:

![image](https://cloud.githubusercontent.com/assets/658107/15891031/8339c0d0-2d6a-11e6-8eff-bd35e29e0568.png)

What happens if you do a full solution clean after you switched to 2.0?

**Comment by DominikPalo** [comment]
After I replaced projects with NuGet packages and cleaned the project, designer works normally, but IntelliSense doens't suggest Material Design element styles.

But when I have toolkit projects included in my solution, cleaning doesn't help (I tried it more times). After every restart of VS and reload of project, designer works until first usage of IntelliSense functionality on the style attribute of some element - after IntelliSense menu is opened, VS freezes for few seconds and the XAML designer crashes.
